repose-aggregator/components/filters/header-normalization-filter/src/main/scala/org/openrepose/filters/headernormalization/HeaderNormalizationFilter.scala

```diff
@@ -83,7 +84,13 @@ class HeaderNormalizationFilter @Inject()(configurationService: ConfigurationSer
         case BlackList => target.headers
       }).foreach(wrappedRequest.removeHeader)
 
-      metricsMeter.mark(s"${target.url.pattern.toString}_${wrappedRequest.getMethod}_request")
+      metricsService foreach {
+        _.createSummingMeterFactory(RequestNormalizationMetricPrefix)
+          .createSummingMeter(MetricRegistry.name(
+            wrappedRequest.getMethod,
+            target.url.pattern.toString.replace('.', '_')))
```

Comment:
Do we maybe want a util method in the metrics service to do this kind of replacement for us?